src/ButtonsServiceProvider.php

```diff
@@ -25,17 +18,18 @@ public function boot()
     {
         $this->loadViewsFrom(__DIR__ . '/resources/views', 'datatables');
 
-        $this->publishAssets();
+        if ($this->app->runningInConsole()) {
+            $this->publishAssets();
 
-        $this->registerCommands();
+            $this->registerCommands();
```

Comment:
I think the command can be excluded from console. AFAIK, there some generators that uses the command and they might using the web interface to do it. This may potentially break those generators.

Reply:
true, reverted this commit.